src/components/Pages/OfficialDocuments/OfficialDocumentPage.js

```diff
@@ -13,7 +13,7 @@ const OfficialDocumentPage = ({ officialDocuments, intl }) => {
 
   const documentsPerPage = 10
   const isMobile = useMobileQuery()
-  const maxPagesShown = isMobile ? 4 : 7 // Desktop : Mobile (pagination pages shown)
+  const maxPagesShown = isMobile ? 5 : 7 // Desktop : Mobile (pagination pages shown)
```

Comment:
def a nitpick and not something that should prevent merging, but this might read a little cleaner and feel a little less magic numbery as:
```
const maxPagesMobile = 5;
const maxPagesDesktop = 7;
const maxPagesShown = isMobile ? maxPagesMobile ? maxPagesDesktop;
```

Reply:
I like that. Changes coming👍